R/pkg/R/SQLContext.R

```diff
@@ -727,6 +730,7 @@ dropTempView <- function(viewName) {
 #' @param source The name of external data source
 #' @param schema The data schema defined in structType
 #' @param na.strings Default string value for NA when source is "csv"
+#' @param ... additional external data source specific named propertie(s).
```

Comment:
ah sorry, I've no idea why this happened - pretty sure I've already corrected that.